requirements.txt

```diff
@@ -1,4 +1,9 @@
-Django==2.2.28
-djangorestframework==3.11.2
-mysqlclient==1.3.9
-pytz==2019.3
+# Pin packages to support and work with py3.6.
+Django==3.1.14
+djangorestframework==3.15.1
+mysqlclient==2.1.1
```

Comment:
Is this still needed if we're using `PyMySQL`?

Reply:
Hmm!.  Thought to keep it(As it is optional, Just in case). It should not create any issues.